test/run_tests.pl

```diff
@@ -28,7 +28,7 @@
     @tests = @ARGV;
 }
 my $list_mode = scalar(grep /^list$/, @tests) != 0;
-if (grep /^alltests|list$/, @tests) {
+if (grep /^(alltests|list)$/, @tests) {
```

Comment:
I'm hoping for this to match for exactly 'list' rather than, for example, 'test_cipherlist' as it does currently. Without this change, `make TEST=test_cipherlist test` runs all tests.

What do you suggest?